examples/event_example/Move.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "event_example"
+version = "0.0.1"
+
+[dependencies]
+#MoveosStdLib = { git = "https://github.com/rooch-network/rooch.git", subdir = "moveos/moveos-stdlib/moveos-stdlib", rev = "main" }
+#MoveosStdLib = { git = "https://github.com/rooch-network/rooch.git", subdir = "moveos/moveos-stdlib/moveos-stdlib", rev = "main" }
+MoveosStdLib = { local = "../../moveos/moveos-stdlib/moveos-stdlib" }
+
+[addresses]
+rooch_examples =  "0xd2d2777904f1c16061308933df89f78fa6ff6966de00ff12d85b52fc631a4621"
```

Comment:
Use `rooch_examples = "_"`, and let the user pass the address by the named address.